name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages_de.properties

```diff
@@ -1681,6 +1681,10 @@ MsgInfoChangingCurrencyNotPossible = Es existieren bereits Buchungen zu diesem W
 
 MsgInfoRetiredSecurities = Inaktive Wertpapiere werden in den Buchungsdialogen nicht (mehr) zur Auswahl angezeigt.\nExistierenden Buchungen werden nicht ge\u00E4ndert.\nAktienkurse werden weiterhin aktualisiert es sei denn das Wertpapier ist mit "kein automatischer Download" konfiguriert.
 
+MsgInvalidImage = Bilddatei fehlerhaft
+
+MsgInvalidImageDetail = Die Datei ''{0}'' konnte nicht als Bild gelesen werden. M\u00F6glicherweise wird das Format nicht unterst\u00FCtzt.
```

Comment:
I think a better translation in german is:
`Die bereitgestellte Datei ''{0}'' scheint kein Bild zu sein oder konnte nicht gelesen werden.`